beamer/tests/agent/unit/test_handlers.py

```diff
@@ -27,6 +28,27 @@
 from beamer.typing import FillId, Termination
 
 
+def get_tx_receipt(status, hashStr) -> TxReceipt:
```

Comment:
Python uses snake_case so `hashStr` should be renamed. Also, we tend to avoid putting types in names.
I think we should just name this variable `tx_hash`, like we have in dozens of other places.